File: integration-tests-java/src/main/java/io/micronaut/jimmer/java/it/repository/BookStoreRepository.java
package io.micronaut.jimmer.java.it.repository;

import io.micronaut.jimmer.java.it.entity.BookStore;
import io.micronaut.jimmer.repository.JRepository;
import io.micronaut.jimmer.repository.annotation.Repository;

@Repository
public interface BookStoreRepository extends JRepository<BookStore, Long> {}

File: integration-tests-kotlin/src/main/kotlin/io/micronaut/jimmer/kotlin/it/repository/BookStoreRepository.kt
package io.micronaut.jimmer.kotlin.it.repository

import io.micronaut.jimmer.kotlin.it.entity.BookStore
import io.micronaut.jimmer.repository.KRepository
import io.micronaut.jimmer.repository.annotation.Repository

@Repository
interface BookStoreRepository : KRepository<BookStore, Long>
